src/overlays/ocaml-static.nix

```diff
@@ -65,14 +65,6 @@ let
       '';
     };
 
-    ocamlgraph = oa: {
-      buildPhase = ''
-        ./configure
-        sed 's/graph.cmxs//' -i Makefile
-        make NATIVE_DYNLINK=false
-      '';
-    };
-
```

Comment:
Instead of removing it, you should gate it on a version. See e.g. `coq-of-ocaml` in this same file